### PR TITLE
Update Simplify.py

### PR DESCRIPTION
**Pull request by malayjoshi13**

### CODE REVIEW

File: server/src/Simplify.py [hunk 1]
```diff
@@ -55,21 +48,19 @@ def get_bert_candidates(input_text, list_cwi_predictions, numb_predictions_displ
...
   return word.lower().strip()
 
-# filename = 'word_to_index.pkl'
-print(os.getcwd())
 filename = open("word_to_index.pkl",'rb')
 word2index = pickle.load(filename)
 sent_max_length = 103
 
-# Function to create the padded sequence
+# 2. Function to create the padded sequence
```
Comment (khareyash05) [inline server/src/Simplify.py:63]: What is the  need of this?

File: server/src/Simplify.py [hunk 2]
```diff
@@ -93,6 +85,8 @@ def complete_missing_word(pred_binary, index_list, len_list):
 
 model_cwi = load_model("model_CWI_full.h5")
 
+#-------------------------------------------------------------------------------------------------------
+
```
Comment (khareyash05) [inline server/src/Simplify.py:89]: need of this? comment

File: server/src/Simplify.py [hunk 3]
```diff
@@ -116,15 +110,9 @@ def simplify_it(input_text):
 
 # ------------------------------------------------------------------------------------------------------- 
 
-# driver code
+# # driver code
```
Comment (khareyash05) [inline server/src/Simplify.py:113]: need of this extra hash

File: server/src/Simplify.py [hunk 4]
```diff
@@ -10,26 +8,21 @@
...
-# Load the BERT  model for masked languge
-bert_model = 'bert-large-uncased'
+# 1. Load the BERT  model for masked languge
+bert_model = 'bert-large-cased-local'
 tokenizer = BertTokenizer.from_pretrained(bert_model)
 model = BertForMaskedLM.from_pretrained(bert_model)
 model.eval()
-# Generate candidates
+
+# 2. Generate candidates
```
Comment (khareyash05) [inline server/src/Simplify.py:25]: need of 2.?

File: server/src/Simplify.py [hunk 5]
```diff
@@ -10,26 +8,21 @@
...
-import os
-print("hello")
-
 
 # Candidates generation using BERT and selection of best candidates based on Zipf values
 
-# Load the BERT  model for masked languge
-bert_model = 'bert-large-uncased'
+# 1. Load the BERT  model for masked languge
+bert_model = 'bert-large-cased-local'
```
Comment (khareyash05) [inline server/src/Simplify.py:20]: if we change the model code might break and may cause errors

**Review by khareyash05 — COMMENTED** [review]
Kindly review the comments

File: server/src/Simplify.py (hunk 1 above)
Comment (malayjoshi13) [reply server/src/Simplify.py:63]: Just merge it. I have tested it.

**Pull request merged**